oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -51,6 +51,13 @@ if __name__ == '__main__':
 
     if storage.lower() == 's3' and not bucket.lower() == 'some_bucket':
         # load s3
+        cmd = 'systemctl start mcs-storagemanager'
```

Comment:
move startup of storagemanager service to this point so we only startup service if we intend to use it